packages/wrangler/src/inspect.ts

```diff
@@ -289,9 +314,9 @@ export default function useInspector(props: InspectorProps) {
 		// each time. Consumers must be destroyed when no longer needed, so create
 		// an abort controller aborted to signal destruction.
 		const sourceMapAbortController = new AbortController();
-		let sourceMapConsumerPromise: Promise<SourceMapConsumer | undefined>;
+		let _sourceMapConsumerPromise: Promise<SourceMapConsumer | undefined>;
```

Comment:
Are we commonly using the `_privateVar` pattern elsewhere? Just want to make sure we are not ad hoc introducing a convention.

Reply:
What is the reason for the concern of it being used directly? Is it exposed outside of `inspect.ts` or you are concerned someone will use it locally in the `inspect.ts` without the getter?